os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -240,8 +240,10 @@ void board_gpio_initialize(void)
 		u32 pinpull;
 	} pins[] = {
 				{PA_23, PIN_INPUT, PullNone},
+				{PA_4, PIN_INPUT, PullNone},
 				/* PB_20 is gpio pin number for LED */
 				{PB_20, PIN_OUTPUT, PullDown},
+				{PB_22, PIN_INPUT, PullNone},
```

Comment:
https://github.com/Samsung/TizenRT/pull/6533
This changing is duplicating, Could you please remove it here?